src/test/java/uk/gov/hmcts/futurehearings/hmi/unit/testing/util/ResponseVerifier.java

```diff
@@ -32,7 +31,7 @@ public static void thenResponseHasErrorForMissingTransactionIDHMCTS(Response res
         assertEquals(2,response.getBody().jsonPath().getMap("$").size());
         Map<String, String> responseMap = response.getBody().jsonPath().getMap("$");
         assertEquals("400",responseMap.get(("Error")));
-        assertEquals("Malformed request. Missing/Invalid property: 'Case Id'",responseMap.get(("Reason")));
+        assertEquals("Malformed request. Missing/Invalid property: 'TransactionIDHMCTS'",responseMap.get(("Reason")));
```

Comment:
Is it part of this change, if not remove this move with the associated ticket pls

Reply:
This test case was failing. So had to fix it.